Test/Utils/ComputeNode/Initialize.ps1

```diff
@@ -2,21 +2,54 @@
 
 . $PSScriptRoot\..\..\PesterLogger\PesterLogger.ps1
 . $PSScriptRoot\..\..\TestConfigurationUtils.ps1
-
+. $PSScriptRoot\Configuration.ps1
+. $PSScriptRoot\Installation.ps1
+. $PSScriptRoot\Service.ps1
 . $PSScriptRoot\..\DockerNetwork\DockerNetwork.ps1
 
 function Initialize-ComputeNode {
     Param (
         [Parameter(Mandatory=$true)] [PSSessionT] $Session,
         [Parameter(Mandatory=$true)] [TestenvConfigs] $Configs,
-        [Parameter(Mandatory=$true)] [Network[]] $Networks
+        [Parameter(Mandatory=$false)] [bool] $PrepareEnv=$true
+    )
+
+    if ($PrepareEnv) {
+        Write-Log "Installing components on testbed..."
+        Install-ComputeServices -Session $Session `
+            -SystemConfig $Configs.System `
+            -OpenStackConfig $Configs.OpenStack `
+            -ControllerConfig $Configs.Controller
+    } else {
+        Write-Log "Not performing environment setup because PrepareEnv flag is set to $PrepareEnv"
+    }
+}
+
+function Clear-ComputeNode {
+    Param (
+        [Parameter(Mandatory=$true)] [PSSessionT] $Session,
+        [Parameter(Mandatory = $true)] [SystemConfig] $SystemConfig,
+        [Parameter(Mandatory=$false)] [bool] $PrepareEnv=$true
     )
 
-    Initialize-ComputeServices -Session $Session `
-        -SystemConfig $Configs.System `
-        -OpenStackConfig $Configs.OpenStack `
-        -ControllerConfig $Configs.Controller
+    if ($PrepareEnv) {
+        Clear-TestConfiguration -Session $Session -SystemConfig $SystemConfig
```

Comment:
While we're at it, can we take a look at `Clear-TestConfiguration`?

It: 

1. cleans up docker networks
2. stops compute node services (but not all!, e.g. doesn't stop node mgr)
3. waits for IP address 

I think we could do something similar like in Initialize-.

```
function Clear-WindowsComputeNode {
    Remove-UnusedDockerNetworks
    Stop-ComputeServices
    Wait-ForIPAddress
    Clear-Logs
    Uninstall-ComputeServices
}
```

The "cleans up docker networks" part we can move to the same location as Initialize-DockerNetworks function maybe?

What I'm suggesting here is proudly presenting what Initialize/Clear logic does in this file, not hiding it in some function in utils somewhere. Of course, we need to keep this file as clear as possible so that it's possible to tell in a glimpse based on function names what exactly will happen.